modules/detect_target/detect_target.py

```diff
@@ -22,6 +23,7 @@ def __init__(self, model_path: str, save_name: str=""):
         self.__model = ultralytics.YOLO(model_path)
         self.__counter = 0
         self.__filename_prefix = save_name + "_" + str(int(time.time())) + "_"
+        self.__device = 0 if torch.cuda.is_available() else "cpu"
```

Comment:
Nit: Move to the line after `self.__model`

Personally, I don't like this format of conditional assignment but that's just my preference, fine to leave as is.

Reply:
Ok in that case we can leave it. Maybe we should just make a note of it in the documentation so people know.